packages/server/core/src/plugins/render/render.ts

```diff
@@ -241,9 +244,42 @@ async function renderHandler(
     },
   };
 
-  let response: Response;
+  let response: Response | null = null;
 
-  if (mode === 'ssr') {
+  const runtimeEnv = getRuntimeEnv();
+
+  const ssrByRouteIds = options.config.server?.ssrByRouteIds;
+
+  if (runtimeEnv === 'node' && ssrByRouteIds && ssrByRouteIds?.length > 0) {
+    const path = await import('node:path');
+    const nestedRoutesJson = await import(
```

Comment:
Why did not import route_spec_file in `@modern-js/plugin-inject-resource` server plugin, then render plugin get it from context.

Reply:
`plugin-inject-resource` does not inject nestedRoute.json, it injects route.json. therefore, after our discussion, I imported it directly here.